tests/e2e_tests/run

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -e
+
+COMPOSE_CMD="docker-compose -p gui-tests \
+             -f ${INTEGRATION_PATH}/docker-compose.yml \
+             -f ${INTEGRATION_PATH}/docker-compose.demo.yml \
+             -f tests/e2e_tests/docker-compose.e2e-tests.yml"
+
+# by default just add minio, with COMPOSE_CMD_BASE this creates the standard onprem ST setup
+COMPOSE_FILES_DEFAULT=( "${INTEGRATION_PATH}/docker-compose.storage.minio.yml" )
+COMPOSE_FILES=()
+
+usage() {
+    echo "runner script for gui-specific integration tests"
+    echo ""
+    echo "./$0"
+    echo -e "\t-h --help"
+    echo -e "\t-c --skip-cleanup \tleave containers running after tests"
+    echo -e "\t-f=<FILE>         \tspecify custom compose file(s); default files will not be used,"
+    echo -e "\t                  \tmake sure to specify all files you need"
+    echo ""
+    echo -e "examples:"
+    echo -e "run all GUI tests, from root of this repo, with default ST setup:"
+    echo -e "GUI_REPOSITORY='<path-to-local-gui-repo>' INTEGRATION_PATH=. ./tests/e2e_tests/$0"
```

Comment:
`$0` will already have the relative or absolute path for the script. It will represent whatever the user has written (for example `./run`, `./tests/e2e_tests/run` of `/home/lluis/full/path/to/repo/tests/e2e_tests/run`.

So, basically, you don't need to add anything here or in line 16.

Reply:
ah thanks, I thought this would just return the filename - should be fixed now...